user_modules/script_runner.lua

```diff
@@ -61,24 +101,55 @@ local function extendedAddTestStep(testStepName, testStepImplFunction, paramsTab
   addTestStep(testStepName, newTestStepImplFunction)
 end
 
---[[ Title + Step approach]]
-local function buildTitle(titleText)
-  local maxLength = 101
-  local filler = "-"
-  local resultTable = {}
-  for line in titleText:gmatch("[^\n]+") do
-    local lineLength = #line
-    if lineLength >= maxLength then
-      table.insert(resultTable, line)
-    else
-      local tailLength = math.fmod(maxLength - lineLength, 2)
-      local emtyLineSideLength = math.floor((maxLength - lineLength) / 2)
-      table.insert(resultTable, filler:rep(emtyLineSideLength) .. line .. filler:rep(emtyLineSideLength + tailLength))
+local function checkTestCompatibility(testCompatibleSdlSettings)
```

Comment:
@aderiabin , please rename function to isTestApplicable() due to returning result is bool.

Reply:
@Itileda  Renamed in 8ec52ec